src/test/shell/integration/startup_options_test.sh

```diff
@@ -82,4 +82,30 @@ function test_autodetect_server_javabase() {
   bazel --noautodetect_server_javabase version &> $TEST_log || fail "Should pass"
 }
 
+# Below are the regression tests for Issue #7489
+function test_multiple_bazelrc_later_overwrites_earlier() {
+  plain_expected_output="INFO: Build completed successfully, 1 total action"
+
+  echo "common --color=yes" > 1.rc
+  echo "common --color=no" > 2.rc
+  bazel "--${PRODUCT_NAME}rc=1.rc" "--${PRODUCT_NAME}rc=2.rc" build &> $TEST_log || fail "Should pass"
```

Comment:
build is a heavyweight command and thus doesn't always work in CI, can you use `info`, `version`, or a similar "simpler" command

Reply:
Makes sense. Done.